app/models/settings/smtp.rb

```diff
@@ -14,14 +14,20 @@ class SMTP < Base
 
     class << self
       def settings
-        return sendgrid_settings if ENV["SENDGRID_API_KEY"].present?
-
-        keys.index_with { |k| public_send(k) }.symbolize_keys
+        if address
```

Comment:
In response to your question in the description, I also don't know whats the bare minimum but should we align this with our ForemInstance.smtp_enabled? method where we check if the the username and password is present?

```

  def self.smtp_enabled?
    (Settings::SMTP.user_name.present? && Settings::SMTP.password.present?) || ENV["SENDGRID_API_KEY"].present?
  end
```

Reply:
that's a good idea. let me do that.

Reply:
updated!